.github/workflows/sync-dependencies.yml

```diff
@@ -0,0 +1,46 @@
+name: Sync Dependencies
+on:
+  pull_request:
+    paths:
+      - 'pyproject.toml'
+      - 'poetry.lock'
+      - '{{cookiecutter.project_slug}}/pyproject.toml'
+      - '{{cookiecutter.project_slug}}/poetry.lock'
+jobs:
+  sync-dependencies:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
+      - name: Sync template dependencies
+        run: |
+          # Extract dependencies from root pyproject.toml template group
+          python3 -c '
+          import toml
```

Comment:
Fix this issue:

Traceback (most recent call last):
  File "<string>", line 2, in <module>
ModuleNotFoundError: No module named 'toml'